sklearn/metrics/tests/test_score_objects.py

```diff
@@ -29,7 +30,7 @@
 from sklearn.svm import LinearSVC
 from sklearn.pipeline import make_pipeline
 from sklearn.cluster import KMeans
-from sklearn.linear_model import Ridge, LogisticRegression
+from sklearn.linear_model import Ridge, LogisticRegression, Perceptron
```

Comment:
`Perceptron` is no longer used. Maybe re-add a test for the multiclass roc AUC on a non-probabilistic multiclass classifier to check that the error message is informative enough?